sklearn/neighbors/tests/test_lof.py

```diff
@@ -89,12 +93,12 @@ def test_lof_values():
     assert_array_almost_equal(-clf2.score_samples([[1.0, 1.0]]), [s_1])
 
 
-def test_lof_precomputed(random_state=42):
+def test_lof_precomputed(global_dtype, random_state=42):
```

Comment:
this test needs to use assert_allclose

Reply:
Currently it's using `assert_array_almost_equal` which should be replaced by `assert_allclose`